src/unistd.rs

```diff
@@ -756,12 +756,29 @@ pub fn close(fd: RawFd) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Reads from a raw file descriptor.
+/// Reads at most as many bytes as will fit into the provided buffer.
```

Comment:
You still need a blank line here after the first line in the docs:
![rendered doc comment](https://user-images.githubusercontent.com/206363/30841638-4808419c-a232-11e7-844a-81c6fc83e2a3.png).

Please add that blank comment line and confirm that only the "Reads from a raw file descriptor" part renders on the `unistd` doc page. You can do this with `cargo doc --no-deps --open`. This also applies to `write()`.

Reply:
Still need a blank line here.